Sources/CocoaLumberjack/include/DDLog.h

```diff
@@ -850,6 +851,7 @@ typedef NS_OPTIONS(NSInteger, DDLogMessageOptions){
 @property (readonly, nonatomic) NSString *threadID; // ID as it appears in NSLog calculated from the machThreadID
 @property (readonly, nonatomic, nullable) NSString *threadName;
 @property (readonly, nonatomic) NSString *queueLabel;
+@property (readonly, nonatomic) NSUInteger qos API_AVAILABLE(macos(10.10), ios(8.0));
```

Comment:
We do also support watchOS and tvOS. So those availability marks should be present as well (also in the `if (@available(...))` statements.

Reply:
Yes but qos_class_self() doesn't define availability for those OSes…

Reply:
Fair point.